src/App.test.jsx

```diff
@@ -0,0 +1,39 @@
+import { render, fireEvent } from '@testing-library/react';
+
+import App from './App';
+
+test('App컴포넌트 초기화 상태', () => {
+  const { container, getByPlaceholderText } = render((
+    <App />
+  ));
+
+  expect(container).toHaveTextContent('To-do');
+  expect(container.hasChildNodes('input')).toBe(true);
+  expect(container.hasChildNodes('button')).toBe(true);
+  expect(container).toHaveTextContent('추가');
+
+  const input = getByPlaceholderText('할 일을 입력해 주세요');
+  expect(input).toHaveAttribute('value', '');
+
+  expect(container.hasChildNodes('list')).toBe(true);
+  expect(container).toHaveTextContent('할 일이 없어요!');
+});
+
+describe('input', () => {
+  it('input에 할 일을 입력하면 입력값으로 바뀐다', () => {
+    const { getByPlaceholderText } = render((
+      <App />
+    ));
```

Comment:
```suggestion
    const { getByPlaceholderText } = render(<App />);
```

이렇게 간단한건 한 줄로 해도 괜찮은 것 같아요.
줄바꿈을 하는 기준이 따로 있으신가요?

Reply:
줄바꿈하는 기준이 따로 있었던 건 아니였어요~ 한 줄로 해도 되는 경우를 잘 생각해보고 불필요한 줄바꿈을 줄이겠습니다 🙏 ~